src/main/java/com/banking/bankservice/service/UserService.java

```diff
@@ -0,0 +1,16 @@
+package com.banking.bankservice.service;
+
+import com.banking.bankservice.model.User;
+import java.util.Optional;
+
+public interface UserService {
+    User create(User user);
+
+    User update(User user);
```

Comment:
see checklist

Reply:
refactored and made only one method `save()`